behat-tests/features/acquia_lift_element_variations.feature

```diff
@@ -1,170 +1,198 @@
-      | machine_name                    | label |
+      | machine_name                    | label                           |
       | testing-campaign-add-variations | Testing campaign add variations |
-    And I am logged in as a user with the "Marketer" role
+    And I am logged in as a user with the "access administration pages,access toolbar,administer visitor actions,manage personalized content" permission
     And I am on the homepage
+    And I am viewing an "Article" content:
+      | title | Test Article Title - Original |
     When I click "Acquia Lift" in the "menu" region
     And I wait for AJAX to finish
     Then I should see the link "Campaigns" in the "lift_tray" region
+
+    # I open the variation set's menu.
     When I hover over "Campaigns" in the "lift_tray" region
     And I click "Testing campaign add variations" in the "lift_tray" region
-    Then I should see the link "Variation Sets" visible in the "lift_tray" region
+    Then I should visibly see the link "Variation Sets" in the "lift_tray" region
     And I should see "0" for the "variation set" count
+
+    # I bring up the "Add variation set" interface.
     When I hover over "Variation Sets" in the "lift_tray" region
     Then I should see the link "Add variation set" in the "lift_tray" region
     Then I should see the link "All variation sets" in the "lift_tray" region
     When I click "Add variation set"
-    And I wait for AJAX to finish
     Then I should see the modal with title "Add a variation set"
     And I should see the link "Webpage elements" in the "modal_content" region
     And I should see the link "Drupal blocks" in the "modal_content" region
     When I click "Webpage elements" in the "modal_content" region
-    And I wait for AJAX to finish
     Then I should not see the modal
-    When I click "#site-name a span" element in the "page_content" region
-    And I wait for AJAX to finish
-    Then I should see the text "<SPAN>" in the "dialog_variation_type" region
+
+    # I add a new variation set.
+    When I click "#page-title" element in the "page_content" region
+    Then I should see the text "<H1>" in the "dialog_variation_type" region
     When I click "Edit text" in the "dialog_variation_type" region
-    And I wait for AJAX to finish
     Then I should not see the variation type dialog
-    And I should see the text "Edit text: <SPAN>" in the "dialog_variation_type_form" region
-    And the "personalize_elements_content" field should contain the site title
-    When I fill in "Test 1" for "personalize_elements_content"
+    And I should see the text "Edit text: <H1>" in the "dialog_variation_type_form" region
+    When I fill in "Test Article Title - Updated 1" for "personalize_elements_content"
     And I fill in "Test variation set" for "title"
-    And I press the "Save" button
-    And I wait for AJAX to finish
+    And I click "#edit-variation-type-submit-form" element in the "dialog_variation_type_form" region
```

Comment:
This seems less intuitive.  Why was it switched to using the element id rather than the button text?